models/eprop_synapse_bsshslm_2020.h

```diff
@@ -606,8 +606,7 @@ eprop_synapse_bsshslm_2020< targetidentifierT >::set_status( const DictionaryDat
     auto it = optimizer_dict->find( names::optimize_each_step );
     if ( it != optimizer_dict->end() )
     {
-      throw BadProperty(
-        "eprop_synapse_bsshslm_2020 only supports optimization in each step optimize_each_step == False." );
+      throw BadProperty( "eprop_synapse_bsshslm_2020 does not have parameter optimize_each_step." );
```

Comment:
Lines 603–610 can be removed, as the error `NESTErrors.DictError: DictError in SLI function Connect_g_g_D_D: Unused dictionary items: optimize_each_step` is already raised earlier.